spec/safe_statements_spec.rb

```diff
@@ -1,18 +1,26 @@
 require "spec_helper"
 
 RSpec.describe PgHaMigrations::SafeStatements do
-  TableLock = Struct.new(:table, :lock_type, :granted)
+  TableLock = Struct.new(:table, :lock_type, :granted, :pid)
   def locks_for_table(table, connection:)
+    identifiers = table.to_s.split(".")
```

Comment:
Will this ever come in with quoting?

Reply:
I don't think so, but we could use `PgHaMigrations::Struct::Table.from_table_name(table)` if we wanted. This is just a test helper though so I don't feel too strongly